### ISSUE TITLE
Cant reach 90.4% precision for 4segments ECOFULL.

### ISSUE THREAD

**Issue by Rayaction** [zhang-can/ECO-pytorch#63]
i use the recommended cmd:
[UCF101 - ECO - RGB] command:

python main.py ucf101 RGB <ucf101_rgb_train_list> <ucf101_rgb_val_list> \
        --arch ECO --num_segments 4 --gd 5 --lr 0.001 --lr_steps 30 60 --epochs 80 \
        -b 32 -i 1 -j 1 --dropout 0.8 --snapshot_pref ucf101_ECO --rgb_prefix img_ \
        --consensus_type identity --eval-freq 1

but the result at 20 epoch is not quite saitisfied.: only 83%. wating to see the prec tommorrow. hope somebody could tell me are there still some tricks? I see a little bit different change in the configuration between this git and that made by MZO.

**Comment by Rayaction** [comment]
used the pretrained model from MZO git which pretrained by K400, trained for a night , waked up and checked its precision, best still around 83%, could u give me some advice?

**Comment by Rayaction** [comment]
ok now i use MZO's code and script and can get 91.89% on 16F, 92.9% on 32F.  
btw guys, the first time i use the code, it cant reach that prec since the dataloader cant get right images from videos.
check this out, it always get the first image. u should calculate the num of frames yourself, and then it works.
secondly, data should be images not videos.
thats all. 
good work mate
